### ISSUE TITLE
.M4V Won't Open - Greyed Out?

### ISSUE THREAD

**Issue by jjmccrae** [griff/metaz#3]
Hey,

I don't know if this is an on going issue, but its seems to be happening a lot recent. I can't seem to open any .M4V files.  I have been using MetaZ for a long time and this has been happening since just before Christmas. You can't drag the file to the app icon, can't drop into the open app, or open from Flie > Open File. All .M4V files are greyed.

This happened, and then I had to restore my MacBook, and it was back to normal, then it appeared again.  Then I sometimes I closed the app and opened it and it worked... not it just won't work.  

Am I the only one who is experiencing this issue? Is it an know issue?  Is there anything I can try to help resolve this??

Thanks.

**Comment by griff** [comment]
We have had some problems like what you describe. A couple of questions:
- First off what is your OS version?
- Have you updated to alpha 9?

Try moving MetaZ away from /Applications and then moving it back (When an app gets copied or moved to /Applications the file types it supports gets registered with Finder)

Could you look in the log via the menu _Help -> View Log_ and make sure this line is there:
`[lvl=2] -[MZPluginController loadedPlugins] Loaded plugin 'org.maven-group.metaz.AtomicParsleyPlugin`

**Comment by griff** [comment]
One more question. Can you open .mp4 files or is it only .m4v files you have problems with? 

Could you also use File -> Open File (you don't have to actually open a file just open the dialog box) and afterwords check the log as described above for the 4 lines found below:

``` text
[lvl=1] -[AppController openDocument:] Found extention mp4
[lvl=1] -[AppController openDocument:] Found extention mov
[lvl=1] -[AppController openDocument:] Found extention qt
[lvl=1] -[AppController openDocument:] Found extention m4v
```

**Comment by jjmccrae** [comment]
Hey sorry for the late reply, been working all weekend and haven't had a chance to get to my computer.

Running Lion 10.7.2,  I have no idea what Alpha 9 is, so probably haven't updated that.

I moved METAZ away from Applications and back, still nothing.   I looked in the View Log for the first line (in the first reply), it is there. 

I can't open any .mp4 either.   Tried the File -> Open File and still nothing, everything is greyed out.  I checked the View Log again for all four 'extensions' and nothing appears in it.

Something I've done wrong??

**Comment by griff** [comment]
Alpha 9 refers to the version of MetaZ like 1.0a9 is version 1.0 alpha 9. I ask because it was released about 2 days ago.

When looking in Preferences -> Plug-ins is the Atomic Parsley plugin enabled? It must be enabled for reading and writing to work and if it is disabled you would get the symptoms you describe.

If the plugin is enabled could you add what is in the log after the "Loaded plugin" lines?

**Comment by jjmccrae** [comment]
Hey,

Thanks for explaining Alpha 9, yeah its the newest version, but this problems was happening on and off since before Christmas, sometimes the files weren't grey out, some times they were. Now they are constantly. (Version 1.0a9 (12.01.20.16)) is deployed in About MetaZ.

Enabled the Atomic Parsley plugin, no idea why it wasn't.  Now I can see that .mp4 can open but not the .m4v. #strange

EDIT ::
Just doubled checked that the .m4v files won't open again before I posted, they were greyed out when I typed the message above, but now they will load.  So so so strange.

Thanks for your help.  It seems to have resolved the problem.
